.github/workflows/cypress.yml

```diff
@@ -0,0 +1,39 @@
+name: Cypress
+on: [deployment_status]
+
+jobs:
+  cypress:
+    name: Run Integration Tests on Chrome / Firefox
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success' && github.event.deployment_status.environment == 'Preview – web-reader'
+    runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:node14.17.0-chrome88-ff89
+      options: --user 1001 # https://github.com/cypress-io/github-action#firefox
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Cypress
+        run: npm i cypress
+
+      - name: Run Default (Electron)
+        uses: cypress-io/github-action@v2
+        with:
+          install: false
+          config: baseUrl=${{ github.event.deployment_status.target_url }}
```

Comment:
Instead of adding the baseUrl to this config object, let's put it in the `env` object like it is in the CPW repo. Adding it this way means that the `baseUrl` value from the cypress.json file will be overwritten in the CI with this `CYPRESS_BASE_URL` value (this is a fancy Cypress thing). Should look like this:

```
      - name: Run Default (Electron)
        uses: cypress-io/github-action@v2
        with:
          install: false
        env:
          CYPRESS_BASE_URL: https://qa-web.openebooks.us
          # Recommended: pass the GitHub token lets this action correctly
          # determine the unique run id necessary to re-run the checks
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

Reply:
For some reason, I was getting a type error (Expect String | Number | Boolean) if I set it as a .env value. 
(Line 38, is this what you mean?)

Reply:
Yes... hmm. I'll look at it tomorrow and see if I can figure out why it wouldn't like that value!!

Reply:
ok. it works! It was my vscode linter that was complaining! Thanks!